index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <script src="data.js"></script>
+  <script defer src="data.js"></script>
```

Comment:
No need to add a `defer` on the data.js file, it doesn't need access to the DOM!